.github/workflows/functional.yml

```diff
@@ -99,19 +98,35 @@ jobs:
         run: |
           git submodule update --init repository-service-tuf-cli
           git submodule foreach git pull origin main
-          pip install -e repository-service-tuf-cli/
+          pip install repository-service-tuf-cli/
```

Comment:
Why did you decide to not install `repository-service-tuf-cli` as an editable installation?
Don't you use a locally available paths?

Reply:
I'm still having problems at this point. 
GH runners don't support editable installation. 
When I install it still not callable from the next step (functional tests). It looks like the ‘rstuf’ is not in the path. I am trying to solve it.

Reply:
Okay, I guess if it's working right now without the editable installation we can leave it like that.